docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,45 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
+
+
+## Using secrets
```

Comment:
Please can you follow the guidelines for headings in the imperative?

The previous heading was Seal a secret / Seal secrets. We need to maintain this terminology

Reply:
👍